mne/io/neuralynx/neuralynx.py

```diff
@@ -0,0 +1,85 @@
+import os.path as op
+from mne.io import BaseRaw
+from mne.io.utils import _read_segments_file
+from mne import create_info
+from mne.utils import logger, verbose, fill_doc
```

Comment:
All imports in `mne/` to other MNE stuff should be relative (other than tests, which should use absolute imports):
```suggestion
from ..base import BaseRaw
from ..utils import _read_segments_file, _mult_cal_one
from ..._fiff.meas_info import create_info
from ...utils import logger, verbose, fill_doc, _soft_import
```